avocado/utils/partition.py

```diff
@@ -250,14 +250,17 @@ def _unmount_force(self, mountpoint):
         :raise PartitionError: On critical failure
         """
         # Human readable list of processes
-        out = process.system_output("lsof " + mountpoint, ignore_status=True)
-        # Try to kill all pids
-        for pid in (line.split()[1] for line in out.splitlines()[1:]):
-            try:
-                process.system("kill -9 %d" % int(pid), ignore_status=True,
-                               sudo=True)
-            except OSError:
-                pass
+        try:
+            out = process.system_output("lsof " + mountpoint, ignore_status=True)
+            # Try to kill all pids
+            for pid in (line.split()[1] for line in out.splitlines()[1:]):
+                try:
+                    process.system("kill -9 %d" % int(pid), ignore_status=True,
+                                   sudo=True)
+                except OSError:
+                    pass
+        except OSError:
+            pass
```

Comment:
Two points here:
1. Shouldn't we at least log (or do anything else with) these exceptions?
2. Why one try..except inside another? 
Suggestion:
If process.system_output() can raise OSError, so:
```python
try:
    process_running = process.system_output(...)  # improving the 'out' var
except OSError:
    log()

if not process_running:
    return

for pid in (...):
    try... except
```